R/html-deps.R

```diff
@@ -24,6 +24,9 @@ createWebDependency <- function(dependency, scrubFile = TRUE) {
   if (!inherits(dependency, "html_dependency"))
     stop("Unexpected non-html_dependency type")
 
+  # Resolve htmlDependencyFunction()
+  dependency <- htmltools::resolveDependencies(list(dependency))[[1]]
```

Comment:
This might be reason enough to export `htmltools:::as_html_dependency()`? https://github.com/rstudio/htmltools/pull/179/files#diff-0766e7ae43e820ca8f914b19eaa87845R163